styles.css

```diff
@@ -34,14 +34,64 @@ main {
   max-width: 88%;
 }
 
+.ham {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+  margin-right: -33px;
+}
```

Comment:
- [ ] Your "X" button is partially visible. Looks like the menu-mobile is overlapping it. You could kindly set a higher z-index for the `.hamburger` class over the menu mobile z-index.
- [OPTIONAL] Your hamburger button is almost being cut out of the screen. Kindly review the margin-right of that button.
![image](https://user-images.githubusercontent.com/61919617/174664909-8c63e7ba-bd3e-4d33-8d6f-c55e12301077.png)